tests/pipeline/stages/testing.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -ex
+
+image_pool_dir="/srv/pool/"
+sudo mkdir -p  "${image_pool_dir}"
+sudo chmod a+rwx "${image_pool_dir}"
+
+__libvirt_dir="/var/run/libvirt/"
+__libvirt_sock="${__libvirt_dir}/libvirt-sock"
+__libvirt_config="${PWD}/tests/pipeline/libvirtd.conf"
+__command_basis="docker run --rm \
```

Comment:
do you mind using $kcli instead of __command_basic, to make reading of those tests easier for people?

Reply:
Makes sense.